integration/commands_freemonitoring_test.go

```diff
@@ -88,17 +90,49 @@ func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
 				Message: `Enumeration value '' for field 'setFreeMonitoring.action' is not a valid value.`,
 			},
 		},
+		"DocumentCommand": {
```

Comment:
Changes in this test might create conflict with the issue community member is working on https://github.com/FerretDB/FerretDB/issues/2735

Reply:
Not sure but if so it won't be hard to resolve them, I think.